Integrations/integration-Jask_v2.yml

```diff
@@ -0,0 +1,1540 @@
+        except ValueError:
+            return_error("Invalid format for date {}, needs to match YYYY-MM-DD".format(dstr))
+        return d
+
+
+    def translate_time_query(relative_time, time_field):
+        """
+        Convert the created or lastSeen argument to querystring
+        """
+        now = datetime.now()
+        if not relative_time or relative_time == 'All time':
+            return ''
+        if relative_time == 'Last week':
+            end = date.today().strftime("%Y-%m-%d")
+            start = datetime.strftime((now - timedelta(days=7)), "%Y-%m-%d")
+            return '{}:{}..{}'.format(time_field, start, end)
+        if relative_time == 'Last 48 hours':
+            end = datetime.now().strftime("%Y-%m-%dT%H:%M:%S")
+            start = datetime.strftime((now - timedelta(hours=48)), "%Y-%m-%dT%H:%M:%S")
+            return '{}:{}..{}'.format(time_field, start, end)
+        if relative_time == 'Last 24 hours':
+            end = datetime.now().strftime("%Y-%m-%dT%H:%M:%S")
+            start = datetime.strftime((now - timedelta(hours=24)), "%Y-%m-%dT%H:%M:%S")
+            return '{}:{}..{}'.format(time_field, start, end)
+
+
+    def add_list_to_q(q, translate):
+        """
+        Add arguments in the translate dictionary to querystring
+        """
+        for v in translate:
+            arg_list = argToList(demisto.getArg(translate[v]))
+            if len(arg_list) == 1:
+                q += " {}:{}".format(v, arg_list[0])
+            elif len(arg_list) > 1:
+                q += " {}:{}".format(v, ','.join(arg_list))
+        return q
+
+
+    def add_status_to_q(q, status):
+        """
+        Convert the status value for the query string
+        """
+        if status != 'not closed':
+            q += " status:{}".format(status)
+            return q
+
+        return q + " -status:closed"
+
+    def add_time_to_q(q, time_field, relative_time=None, time_from=None, time_to=None):
+        """
+        Add the time filter to the query string for created or lastSeen
+        time_field_types = ['created', 'lastSeen']
+        Defaults to All time if no fields specified
+        """
+        if relative_time:
+            if time_from or time_to:
+                return_error('You cannot specify absolute times [time-to, time-from] ' +
+                    'with relative time [{}]'.format(time_field))
+            else:
+                if translate_time_query(relative_time, time_field) != '':
+                    q += translate_time_query(relative_time, time_field)
+        elif time_from and time_to:
+            q += " {}:{}..{}".format(time_field, convert_string_date_to_date(time_from), convert_string_date_to_date(time_to))
+        elif time_from or time_to:
+            return_error('You must specify both absolute times [time-to, time-from] ' +
+                'or relative time [{}]'.format(time_field))
+
+        return q
+
+
+    def get_insight_from_context(context):
```

Comment:
No need to search for it in the context, we have a mechanism to handle that, please have look [here](https://github.com/demisto/content/tree/master/docs/DT)